doc/advanced/override-default-entrypoint.rst

```diff
@@ -0,0 +1,41 @@
+Overriding the default entry point
+----------------------------------
+
+In some cases, it may be necessary to override the default main entry point of
+an Ocaml program. For example, this is the case if you want to let your program
+handle argument wildcards expansion on Windows.
+
+Fortunately, the ``foreign_stubs`` stanza can be leveraged for that usage.
+
+Let's consider a ``hello.ml`` file and the following ``main.c`` file that
+redefines the entry point:
+
+.. code:: C
+
+    #include <stdio.h>
+
+    #include "caml/mlvalues.h"
+    #include "caml/callback.h"
+
+    CAMLextern void caml_do_exit (int);
+
+    int main_os(int argc, char_os **argv)
```

Comment:
It can be clearer to add a comment to specify that this is the entry point you are trying to redefine.

Reply:
I agree, reading this I was confused where the redefinition happens and only then guessed that it happens because `main_os` is called this. I think it would be nicer to spell that out somewhere explicitely.

Reply:
Better, thanks for the update :)